app_dart/lib/src/request_handlers/refresh_github_commits.dart

```diff
@@ -49,23 +50,31 @@ class RefreshGithubCommits extends ApiRequestHandler<Body> {
     @visibleForTesting this.datastoreProvider = DatastoreService.defaultProvider,
     @visibleForTesting this.httpClientProvider = Providers.freshHttpClient,
     @visibleForTesting this.gitHubBackoffCalculator = twoSecondLinearBackoff,
+    TabledataResourceApi tabledataResourceApi,
   })  : assert(datastoreProvider != null),
         assert(httpClientProvider != null),
         assert(gitHubBackoffCalculator != null),
+        tabledataResourceApi = tabledataResourceApi ?? TabledataResourceApi,
```

Comment:
This looks strange - shouldn't it be `TabledataResourceApi()`?

Why aren't we assigning this in the constructor the way the others are?

Reply:
Please refer rewritten API.